changelog-2022.12.18-12.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index 24a0490187..22f110f920 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -107,6 +107,22 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2022.12.18-12" date="2022-12-18">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>New OpenStreetMap data as of December 16, 2022</li>
```

Comment:
• New OpenStreetMap data as of December 16, 2022
• Improved routing algorithm
• Do not route over highway=construction :)
• Support multiple track geometry in KML files
• Fixed KML 2.3 track import from OpenTracks
• Faster parsing and rendering of complex KML tracks
• Allow adding some shops, galleries, music schools in Editor
• Improved city borders in search results

Map Styles:
• Improved, less bright icons in Night mode
• Fixed subway icons in some cities
• Properly draw barriers for areas and relations
• Fixed duplicated names on the world map
• Updated cave, mountain pass, peak, volcano, emergency phone, historic ship, ice cream, nightclub, music shop, stadium, video games icons
• Fixed wrong priorities of leisure=pitch, historic, post office
• More consistent filled/round icons (round means some kind of business/service)

Translations:
• TTS (Text-To-Speech) for Brazil, Mexican Spanish and Norwegian
• Fixed German and Hungarian search strings
• Updated Czech, Spanish translations

Reply:
Done